neo_batterylevelshutdown/usb.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import os
+import subprocess
+from distutils.dir_util import copy_tree
+
+
+
+class USB:
+
+    def __init__(self):
+        pass
+
+    def isUsbPresent(self, devPath='/dev/sda1'):
+        '''
+        Returns if there is a USB plugged into specified devPath
+        :return: True / False
+        '''
+        logging.debug("Checking to see if usb is mounted")
+        return os.path.exists(devPath)
+
+
+    def unmount(self, curPath='/media/usb0'):
+        '''
+        Unmount the USB drive from curPath
+        :return:  True / False
+        '''
+        try:
+            logging.debug("Unmounting file at location {}".format(curPath))
+            response = subprocess.call(['umount', curPath])  # unmount drive
+            return True if response == 0 else False
+        except:
```

Comment:
Please be specific about the exception that is being caught.

Reply:
This has been addressed